src/components/withHeroChart.js

```diff
@@ -73,19 +73,19 @@ const withHeroChart = (ComposedComponent) => {
           },
         },
         tooltip: {
-          enabled: true,
+          enabled: false,
           shared: true,
           crosshairs: true,
           borderRadius: 8,
           headerFormat: '<small>{point.key}</small><table>',
           pointFormatter: function() {
             // this refers tp "Point"
             const {units} = this.series.options;
-            const key = symbolChars[this.series.symbol];
+            const symbol = symbolChars[this.series.symbol];
             const value = `${units === '$' ? '$' : ''}${this.y}${units === '%' ? '%' : ''}`;
 
             return `<tr>
-                      <td><span style="font-size:20px; color: ${this.series.color}">&#${key}</span></td>
+                      <td><span style="font-size:20px; color: ${this.series.color}">&#${symbol}</span></td>
```

Comment:
i copy pasted the unicode glyphs https://github.com/govau/datavizkit/pull/142/files#diff-06dc5344fa05bc0eecd1c7d3078158cfR6 What do you think?